integration-tests/initiateDataRequest.spec.ts

```diff
@@ -116,7 +117,6 @@ describe('Submit a PII request with approved ticket data', () => {
       assertEventPresent(initiateDataRequestEvents, DATA_SENT_TO_QUEUE_MESSAGE)
 
       const messageId = getQueueMessageId(initiateDataRequestEvents)
-      console.log('messageId', messageId)
```

Comment:
I think the message id is useful for debugging in the logs as it's used to link more than one log stream.